plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -1454,7 +1457,7 @@ fu_synaptics_mst_device_rescan(FuDevice *device, GError **error)
 	/* this is not a valid customer ID */
 	if ((self->board_id >> 8) == 0x0) {
 		fu_device_remove_flag(device, FWUPD_DEVICE_FLAG_UPDATABLE);
-		fu_device_set_update_error(device, "cannot update as CustomerID is unset");
+		g_debug("%s: cannot update as CustomerID is unset", fu_device_get_name(device));
```

Comment:
Not sure I get this. Isn't setting the update error entirely appropriate? Maybe an inhibit might be better?

Reply:
Having the error means that it shows up in `get-devices` output, but users can't do anything about it.  Yeah I think inhibit will work fine here too.

Reply:
I thought we only showed them in get-updates if they were updatable unless we did the `--show-all` thing?

Reply:
Nope - take a look at the logic: https://github.com/fwupd/fwupd/blob/master/src/fu-util-common.c#L251

I think inhibiting will help it though.